keras_cv/backend/random.py

```diff
@@ -15,6 +15,9 @@
 from keras_cv.backend.config import multi_backend
 
 if multi_backend():
-    from keras_core.random import *  # noqa: F403, F401
+    try:
+        from keras.random import *  # noqa: F403, F401
+    except ImportError:
+        from keras_core.random import *  # noqa: F403, F401
 else:
-    from keras_core.src.backend.tensorflow.random import *  # noqa: F403, F401
+    from tensorflow.random import *  # noqa: F403, F401
```

Comment:
These seems dangerous, and not what we had before. `tensorflow.random` is not the same API surface as `keras.random` or `keras_core.random` (which are the same).

We should probably do something like this
https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/backend/random.py

That is, keras_core is the "compat shim" for the random API on Keras 2. On Keras 3 we just use `keras.random` directly.

Reply:
done